packages/nextjs/src/server/clerkMiddleware.ts

```diff
@@ -37,6 +38,10 @@ type ClerkMiddlewareHandler = (
 ) => NextMiddlewareReturn;
 
 export type ClerkMiddlewareOptions = AuthenticateRequestOptions & { debug?: boolean };
+export type EnforcedClerkMiddlewareOptions = {
```

Comment:
Same comment about `EnforcedClerkMiddlewareOptions` - we should't rely on types like these as we've found that they are not maintainable in the long run. Can we rely on TS inference more within the functions?

Also let's be cautious about the `export` keyword as this doesn't look like a type we want to make public.

Reply:
This has been removed.